data/misc/mapping_temp.csv

```diff
@@ -0,0 +1,3221 @@
+FIPS,HSA,hsa_name
+1001,171,"Montgomery (Montgomery), AL - Autauga, AL"
```

Comment:
Is this `mapping_temp.csv` file meant to be checked in?

Reply:
nope it is not :( thanks